src/infrastructure/container.py

```
@@ -0,0 +1,96 @@
+            descriptor.instance = instance
+
+        return instance
+
+    def set_config(self, key: str, value: Any) -> None:
+        self._config[key] = value
+
+    def get_config(self, key: str, default: Any = None) -> Any:
+        return self._config.get(key, default)
+
+
+def inject(container: Container):
```

Ruff (0.12.2)

Missing return type annotation for public function `inject`

(ANN201)